packages/xo-web/src/common/intl/messages.js

```diff
@@ -2401,6 +2401,8 @@ const messages = {
   checkProxyHealth: 'Test your proxy',
   proxyTestSuccess: 'Test passed for {name}',
   proxyTestSuccessMessage: 'The proxy appears to work correctly',
+  proxyTestFailed: 'Test failed for {name}',
+  proxyTestFailedMessage: 'Unable to connect to this proxy',
```

Comment:
This is not a generic error message, please name this message correctly.

Reply:
I don't see what do you mean, it's a message when the proxy test fails. Please see the naming in case of a successful test https://github.com/vatesfr/xen-orchestra/blob/9132611aa3d63a1e992c1d367ff51a6ffe854e91/packages/xo-web/src/common/intl/messages.js#L2403

Reply:
Unless I'm mistaken it's an error message when the proxy test fails *due to a network error*.